ServiceBased/ProAddIn/WorkflowManagerSampleAddIn/Module1.cs

```diff
@@ -0,0 +1,47 @@
+﻿using ArcGIS.Core.CIM;
+using ArcGIS.Core.Data;
+using ArcGIS.Core.Geometry;
+using ArcGIS.Desktop.Catalog;
+using ArcGIS.Desktop.Core;
+using ArcGIS.Desktop.Editing;
+using ArcGIS.Desktop.Extensions;
+using ArcGIS.Desktop.Framework;
+using ArcGIS.Desktop.Framework.Contracts;
+using ArcGIS.Desktop.Framework.Dialogs;
+using ArcGIS.Desktop.Framework.Threading.Tasks;
+using ArcGIS.Desktop.Layouts;
+using ArcGIS.Desktop.Mapping;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace WorkflowManagerSampleAddIn
+{
+    internal class Module1 : Module
+    {
+        private static Module1 _this = null;
+
+        /// <summary>
+        /// Retrieve the singleton instance to this module here
+        /// </summary>
+        public static Module1 Current => _this ??= (Module1)FrameworkApplication.FindModule("WorkflowManagerSampleAddIn_Module");
+
+        #region Overrides
+        /// <summary>
+        /// Called by Framework when ArcGIS Pro is closing
+        /// </summary>
+        /// <returns>False to prevent Pro from closing, otherwise True</returns>
+        protected override bool CanUnload()
+        {
+            //TODO - add your business logic
+            //return false to ~cancel~ Application close
```

Comment:
Should this comment be removed?

Reply:
Similar comment about boilerplate comments from the addin template. I think we can leave as-is for the user to update as needed.